subsys/bluetooth/audio/Kconfig.tbs

```
@@ -152,4 +152,63 @@ config BT_DEBUG_TBS
 
 endif # BT_TBS
 
+
+##################### Call Control Client #####################
+
+config BT_TBS_CLIENT
```

Should this be a menuconfig instead, in order to give a better grouping in `menuconfig` with a subpage for all TBS_CLIENT related settings

```suggestion
menuconfig BT_TBS_CLIENT
```

---

For now I'd like to keep it as is, as that will follow the way the other LE Audio Kconfig files are defined. 

If we want to use `menuconfig` for any of them, we should do it for all of them as they follow the same pattern.